delivery_gls_asm/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery GLS-ASM",
+    "summary": "Delivery Carrier implementation for GLS with ASMRed API",
+    "version": "12.0.1.0.0",
+    "category": "Delivery Carrier",
```

Comment:
This category doesn't exist. Not sure if we should create it if the rest don't use it. `delivery` module is in "Stock".